go/vt/wrangler/testlib/emergency_reparent_shard_test.go

```diff
@@ -244,22 +235,12 @@ func TestEmergencyReparentShardPrimaryElectNotBest(t *testing.T) {
 	defer newPrimary.StopActionLoop(t)
 
 	// old primary, will be scrapped
-	oldPrimary.FakeMysqlDaemon.ReplicationStatusError = mysql.ErrNotReplica
+	oldPrimary.FakeMysqlDaemon.ReplicationStatusError = fmt.Errorf("old primary stopped working")
```

Comment:
👍

Reply:
The remainder of the test was written with the expectation that the oldPrimary was down, but after the bug fix, the test started failing. So I changed the returned error which is commensurate to the primary failing